UChicago VuFind.js

```diff
@@ -822,6 +812,95 @@ var testCases = [
 		"type": "web",
 		"url": "https://catalog.lib.uchicago.edu/vufind/Search/Results?lookfor=andre+gide&type=AllFields",
 		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://catalog.lib.uchicago.edu/vufind/Record/10773190",
+		"items": [
+			{
+				"itemType": "thesis",
+				"title": "Characterizing kinds: A semantics for generic sentences",
+				"creators": [
+					{
+						"firstName": "Matthew",
+						"lastName": "Teichman",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "University of Chicago",
+						"creatorType": "contributor",
+						"fieldMode": true
+					}
+				],
+				"date": "2015",
+				"abstractNote": "In this text, I argue that generic statements---statements of the form Fs are G, such as 'Bears are furry'---are particular statements about kinds, rather than general statements about individual objects. Although statements of this form intuitively seem like generalizations, I claim that in this case, appearances are deceptive. First, I present new linguistic evidence which raises problems for the standard quantificational theory of generic sentences, according to which generic sentences contain a hidden, unpronounced quantifier. Though the simple kind theory has served as a standard alternative to quantificational approaches in the literature on generics since Carlson (1977), it also has a more sophisticated cousin, which has largely been ignored. I develop an extension of the sophisticated kind theory and show how it can neatly account for these phenomena while sidestepping the standard objections to the simple kind theory. At a broader level, I would like to claim that if a kind theory provides the best explanation for the truth conditions of these sentences in English, then it tells us something interesting about English speakers: namely, that in virtue of their speaking English, they implicitly presuppose an ontology with kinds as possible objects. In this way, I suggest, the search for the best semantic theory of generic sentences has the potential to lead us towards a new, philosophically valuable conception of kindhood",
+				"archive": "ProQuest Dissertations & Theses,",
+				"libraryCatalog": "UChicago VuFind",
+				"numPages": "1",
+				"place": "Ann Arbor",
+				"shortTitle": "Characterizing kinds",
+				"university": "University of Chicago.",
```

Comment:
OK, really, really last thing. Can we strip the period off the end? To avoid messing with acronyms ("U.C.L."), how about:

```js
if (item.university) {
	item.university = item.university.replace(/([a-z])\.\s*$/, '$1');
}

Reply:
Actually, done :)